galleon-plugins/src/main/java/org/wildfly/galleon/plugin/WfInstallPlugin.java

```diff
@@ -1066,7 +1113,11 @@ private void extractSchemas(Path moduleArtifact) throws IOException {
         }
     }
 
-    private boolean requireChannel(ProducerSpec spec) {
+    boolean requireChannel(String artifact_ga) {
```

Comment:
nitpick `s/artifact_ga/artifactGA` 

(AFAICT other parameters use camel case not underscore)